### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-ansible-role-with-test-user.git`](https://github.com/cisagov/skeleton-ansible-role-with-test-user.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/ansible-role-cobalt-strike.git ansible-role-cobalt-strike
    cd ansible-role-cobalt-strike
    git remote add skeleton https://github.com/cisagov/skeleton-ansible-role-with-test-user.git
    git remote set-url --push skeleton no_push
    git switch develop
    git switch --create lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add meta/main.yml molecule/default/molecule.yml molecule/default/requirements.yml 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Confirm each item in the "Pre-approval checklist" below.

1. Remove any of the checklist items that do not apply.

1. Ensure every remaining checkbox has been checked.

1. Mark this draft pull request "Ready for review".

## ✅ Pre-approval checklist ##

Remove any of the following that do not apply. If you're unsure about
any of these, don't hesitate to ask. We're here to help!

- [x] ✌️ The conflicts in this pull request have been resolved.
- [x] All relevant type-of-change labels have been added.
- [x] All new and existing tests pass.

---

> [!NOTE]
> You are seeing this because one of this repository's maintainers has
> configured [Lineage] to open pull requests.

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

**Comment by jsf9k** [comment]
The failing ARM64 builds are because `TeamServerImage` is an x86_64 binary:
```console
root@debian11-systemd-arm64:/opt/cobaltstrike# ./teamserver

[*] Checking TeamServerImage for local update

[*] Verifying MD5 Message Digest for TeamServerImage
TeamServerImage: OK

[*] Will use existing X509 certificate and keystore (for SSL)

[*] Starting teamserver
./teamserver: line 57: ./TeamServerImage: No such file or directory
root@debian11-systemd-arm64:/opt/cobaltstrike# cat -n teamserver | tail 
    48          keytool -keystore ./cobaltstrike.store -storepass 123456 -keypass 123456 -genkey -keyalg RSA -alias cobaltstrike -dname "CN=Major Cobalt Strike, OU=AdvancedPenTesting, O=cobaltstrike, L=Somewhere, S=Cyberspace, C=Earth"
    49  fi
    50  
    51  # (pre-4.6) start the team server.
    52  # java -XX:ParallelGCThreads=4 -Dcobaltstrike.server_port=50050 -Dcobaltstrike.server_bindto=0.0.0.0 -Djavax.net.ssl.keyStore=./cobaltstrike.store -Djavax.net.ssl.keyStorePassword=123456 -server -XX:+AggressiveHeap -XX:+UseParallelGC -classpath ./cobaltstrike.jar server.TeamServer $*
    53  
    54  # 4.6+ start the peclone process.
    55  # TODO: Does this need "-server" option?
    56  print_section_header "Starting teamserver"
    57  ./TeamServerImage -Dcobaltstrike.server_port=50050 -Dcobaltstrike.server_bindto=0.0.0.0 -Djavax.net.ssl.keyStore=./cobaltstrike.store -Djavax.net.ssl.keyStorePassword=123456 teamserver $*
root@debian11-systemd-arm64:/opt/cobaltstrike# file TeamServerImage
TeamServerImage: ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, BuildID[sha1]=9fe9174f4bcbdab611ed28f984a98c354b480dd0, for GNU/Linux 3.2.0, with debug_info, not stripped
```

**Review by dv4harr10 — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -169,12 +169,50 @@ jobs:
...
+          #
+          # [*] Will use existing X509 certificate and keystore (for SSL)
+          #
+          # [*] Starting teamserver
+          # ./TeamServerImage: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by ./TeamServerImage)
+          # - debian10-systemd
+          - debian11-systemd
+          - debian12-systemd
+          - debian13-systemd
+          # - fedora39-systemd
```
Comment (mcdonnnj) [inline .github/workflows/build.yml:211]: Just as a reminder for why these are disabled.

```suggestion
          # # Only Debian-based platforms are currently supported.
          # - fedora39-systemd
```

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -169,12 +169,50 @@ jobs:
...
+          # - fedora40-systemd
+          # - kali-systemd
```
Comment (mcdonnnj) [inline .github/workflows/build.yml]: Why is Kali disabled?

**Review by mcdonnnj — APPROVED** [review]

File: .github/workflows/build.yml (hunk 2 above)
Comment (jsf9k) [reply .github/workflows/build.yml]: That was a mistake on my part.  Please see commit 8293462.

**Review by dav3r — APPROVED** [review]

**Pull request merged**